packages/client/src/components/FreeFooter.svelte

```diff
@@ -13,6 +13,7 @@
 
 <style>
   .free-footer {
+    min-height: 51px;
```

Comment:
This previously flip-flopped in height as fonts loaded in, setting it to a constant min of 51px fixes this.